src/HDInsight/HDInsight/Az.HDInsight.psd1

```diff
@@ -107,8 +107,7 @@ CmdletsToExport = 'Get-AzHDInsightJob', 'New-AzHDInsightSqoopJobDefinition',
 
 # Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
 AliasesToExport = 'Enable-AzHDInsightOMS', 'Disable-AzHDInsightOMS', 
-               'Add-AzHDInsightConfigValues', 'Get-AzHDInsightOMS', 
```

Comment:
Deprecate cmdlets is not expected to be removed in breaking changes notice release. List them in the release notes.

Reply:
> Deprecate cmdlets is not expected to be removed in breaking changes notice release. Please list all the breaking changed cmdlets in the release notes.

Hi @msJinLei , Do you think that we should remove the outdated alias and breaking change notice?  As you see, The cmdlet  `Add-AzHDInsightConfigValues `  has a breaking change notice said that its alias will be removed at version 2.0.0. However now the stable version is 2.0.1, so I think that the alias should be removed.

Reply:
> > Deprecate cmdlets is not expected to be removed in breaking changes notice release. Please list all the breaking changed cmdlets in the release notes.
> 
> Hi @msJinLei , Do you think that we should remove the outdated alias and breaking change notice? As you see, The cmdlet `Add-AzHDInsightConfigValues ` has a breaking change notice said that its alias will be removed at version 2.0.0. However now the stable version is 2.0.1, so I think that the alias should be removed.
> 
> The cmdlet `Get-AzHDInsightProperties` has the same problem.

Have added back.